### DISCUSSION TITLE
Web Wallet questions

### DISCUSSION THREAD

**Discussion by HereZesaru** [walt-id/.github#86] [Help]
https://github.com/walt-id/waltid-identity/tree/main/waltid-applications/waltid-web-wallet  I need to create a web wallet on a webpage, based on your documentation it talks about docker and bun to run the project. Do I have to use Bun to create my wallet? Can I run the project using other platforms or standard web development kits instead of Bun or Docker? Also, I see your wallet uses Vue. Do we have to create our wallet in Vue, is it the only platform?

**Comment by alegomes** [comment]
This forum is no longer being monitored. Please, if you still need our help, submit your question as a [Community Stack issue](https://github.com/walt-id/waltid-identity/issues).

All the best
